src/imitation/data/types.py

```diff
@@ -168,12 +351,16 @@ def transitions_collate_fn(
         info dict into a single dict, which is incorrect.)
     """
     batch_no_infos = [
```

Comment:
This variable name is now a bit of a misnomer (it now also doesn't include observations).